packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -554,6 +555,22 @@ class _AppBarState extends State<AppBar> {
         overflow: TextOverflow.ellipsis,
         child: title,
       );
+
+      // Set maximum text scale factor to [_kMaxTitleTextScaleFactor] for the
```

Comment:
This comment doesn't help developers reading the API docs however it's an odd enough corner case that I think it's OK to leave it as it is - a private implementation comment.